Renders absolutely nothing when header is not enabled.

t123yh opened the pull request:
# Description
When downloading blob data from database, the header template will be rendered as `\n` which adds an extra 0x0A at the beginning of the blob data. This fixes the problem.

williamdes commented:
@t123yh thank you for the pull-request, I need to reproduce it and check the fix

Commit:
Renders absolutely nothing when header is not enabled.

When downloading blob data from database, the header template will be rendered as `\n` which adds an extra 0x0A at the beginning of the blob data. This fixes the problem.

Signed-off-by: Yunhao Tian <18373444@buaa.edu.cn>

williamdes committed:
Add a test for pull-request #15666

Signed-off-by: William Desportes <williamdes@wdes.fr>

williamdes commented:
Thank you @t123yh for your contribution !

I had to do some changes to your repo since you used master branch and not QA_5_0 as a base
If you want to contribute more I would recommend you do delete and re-fork the repository
